src/reducers/widgets/index.js

```diff
@@ -0,0 +1,28 @@
+import { CLOCK_TICK } from '../../actions/widgets'
+import clockTicked from './clockReducer'
+
+import dummyClock from './dummy-data/dashboard-tools/clock'
+
+const updateWidget = ( state, action, type, updater ) => {
+  let widgetIndex = 0
```

Comment:
Anytime I see the declaration of a counter like variable outside of a loop, I reach for functional methods on collection objects. Also, `widgetIndex` is a little verbose - there is no need to disambiguate in this context...

```js
const index = state.indexOf( widget => widget.type === type )
```